app/decorators/guest_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class GuestDecorator
+  def lockout_time_remaining_in_words
```

Comment:
It's not necessary to have separate logic when there is no user. There is only one scenario. The countdown always needs to based on the time at which the user was locked before they were signed out. That's because if the user tries to sign back in while they are still locked out, we want to show them how much time they have left before they can log back in. This made me realize that the order is wrong in sessions controller as well. We need to capture the info before the users signs out, then sign them out, then render the partial.

What I had in mind was this:
```ruby
lockout_time_in_words = current_user.decorate.lockout_time_remaining_in_words
lockout_time_remaining = current_user.decorate.lockout_time_remaining

sign_out

render 'two_factor_authentication/shared/max_login_attempts_reached', 
       locals: { 
         type: type, 
         lockout_time_in_words: lockout_time_in_words, 
         lockout_time_remaining: lockout_time_remaining
       }
```
It would be nice to be able to extract this into a module to be shared in both places, but the problem is when they try to sign back in, we don't know which 2FA method caused them to be locked out, so in the sessions controller, we'll need a generic message. Currently, it's hardcoding the `type` to `otp`, which can be misleading.